tasks/go.py

```diff
@@ -408,10 +352,31 @@ def generate_licenses(ctx, filename='LICENSE-3rdparty.csv', verbose=False):
     print("licenses files generated")
 
 
+# FIXME: This doesn't include licenses for non-go dependencies, like the javascript libs we use for the web gui
```

Comment:
Added this FIXME because I realized this is incomplete, but it's unrelated to my changes.